src/editor/editor.cpp

```diff
@@ -265,7 +265,12 @@ Editor::update(float dt_sec, const Controller& controller)
     m_particle_editor_request = false;
     std::unique_ptr<Screen> screen(new ParticleEditor());
     if (m_particle_editor_filename)
-      static_cast<ParticleEditor*>(screen.get())->open("particles/" + *m_particle_editor_filename);
+    {
+      // realpath() is necessary for particle files outside the particles/ folder
+      std::string path = physfsutil::realpath("particles/" + *m_particle_editor_filename);
```

Comment:
Hey, whatever happened to `FileSystem::join`? Or `std::filesystem::path::operator/`? Not the only place where this happens so if you're gonna fix this then do it there aswell

Reply:
Similarly to above, I have opted not to change the code more than necessary to keep the changes minimal and avoid going against eventual preferences. The join function also does not normalize the path, so it would be an extra call rather than a replacement.